diff --git a/pkg/agent/as3/postManager.go b/pkg/agent/as3/postManager.go
index 3d8e6b99..c055ed84 100644
--- a/pkg/agent/as3/postManager.go
+++ b/pkg/agent/as3/postManager.go
@@ -139,6 +139,7 @@ func (postMgr *PostManager) postConfig(data string, tenants []string) (bool, str
 		log.Errorf("[AS3] Creating new HTTP request error: %v ", err)
 		return false, responseStatusCommon
 	}
+	log.Debugf("[AS3] AS3 Declaration: %v", data)
 	log.Debugf("[AS3] posting request to %v", cfg.as3APIURL)
 	req.SetBasicAuth(postMgr.BIGIPUsername, postMgr.BIGIPPassword)

	 
